torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2833,11 +2850,15 @@ Tensor as_strided_backward(
 
   // Step (1): create underlying tensor as "storage"
   auto shared_offset =
-      std::min(input_geometry.storage_offset(), storage_offset);
+      // TODO: symint-ify. Do we need a min() and max() for SymInts?
```

Comment:
Yes, std::min/max working would be good. Can be implemented with guard or plumbing all the way to sympy